README.md

```diff
@@ -27,6 +27,7 @@ Check out the examples:
 * [Curved lines and self links](https://vasturiano.github.io/force-graph/example/curved-links/) ([source](https://github.com/vasturiano/force-graph/blob/master/example/curved-links/index.html))
 * [Text in links](https://vasturiano.github.io/force-graph/example/text-links/) ([source](https://github.com/vasturiano/force-graph/blob/master/example/text-links/index.html))
 * [Dash odd links](https://vasturiano.github.io/force-graph/example/dash-odd-links/) ([source](https://github.com/vasturiano/force-graph/blob/master/example/dash-odd-links/index.html))
+* [Dash odd links (declaratively via linkLineDash)](https://vasturiano.github.io/force-graph/example/dash-odd-links-declarative/) ([source](https://github.com/vasturiano/force-graph/blob/master/example/dash-odd-links-declarative/index.html))
```

Comment:
I think it's ok to just replace the existing "dash odd links" example now we have this functionality built-in.

Reply:
Replaced